tc-health-client/config/config.go

```diff
@@ -80,12 +82,38 @@ func (lcfg LogCfg) InfoLog() log.LogLocation    { return log.LogLocation(lcfg.Lo
 func (lcfg LogCfg) DebugLog() log.LogLocation   { return log.LogLocation(lcfg.LogLocationDebug) }
 func (lcfg LogCfg) EventLog() log.LogLocation   { return log.LogLocation(log.LogLocationNull) } // not used
 
-func ReadCredentials(cfg *Cfg) error {
-	fn := cfg.TOCredentialFile
-	f, err := os.Open(fn)
+/**
+ * cfg - the existing config
+ * updating - when true, existing credentials may be updated from the credential file
+ */
```

Comment:
This godoc should start with `ReadCredentials`.

Reply:
@zrhoffman change made